### DISCUSSION TITLE
Question about clozes

### DISCUSSION THREAD

**Discussion by j-r-e-i-d** [ObsidianToAnki/Obsidian_to_Anki#323] [Q&A]
I suspect I'm really stupid. According to the wiki "The 'CurlyCloze' option will apply to any note type that has "Cloze" in the name"
I can't get Cloze deletions to work. 
Can someone walk me through it? Ideally I'd like to have them interspersed with my normal questions. I am separating my questions with one area per file eg: questions_chemistry.md, questions_anatomy.md

Thanks

**Comment by saravanasmailbox** [comment]
Fine name doesn't matter. Inside you files ( questions_chemistry.md, questions_anatomy.md) use below instructions to create a cloze.

Do these, it will work
START
Cloze
This is a {cloze} note with {two clozes}
Tags: Testing

END

STARTI [Cloze] This is a another {cloze} note with {two clozes}   ENDI
